### ISSUE TITLE
PE Import Table Issue

### ISSUE THREAD

**Issue by 007basaran** [czs108/Windows-PE-Packer#6]
Hello Brother,

First of all, thanks for sharing the project.

I embed an extra dll file with STUD_PE in any .exe file, after this embedding process, when I pack it with packer, the embedded file disappears and loses its function.

When I encrypt the relevant .exe file without adding an extra dll file, and when I add the DLL file later, this time the file cannot access the RAM and the program closes.

In short, what needs to be resolved is that any .dll file transferred and embedded in the Import Table with STUD_PE needs to be detected and included by the packer. 

Kind Regards.
